src/devices/llama_devices.ts

```diff
@@ -0,0 +1,165 @@
+    name: 'switch_actions',
+    description: 'Command to be generated when the switch moves between its two states',
+    values: [
+        {
+            name: 'on/off',
+            value: 0,
+        },
+        {
+            name: 'off/on',
+            value: 1,
+        },
+        {
+            name: 'toggle',
+            value: 2,
+        },
+    ],
+};
+const configs: Config[] = [
+    switchTypeConf,
+    switchAtionConf,
+];
+
+function mapModernActionEnum() {
+    return modernExtend.actionEnumLookup({
+        actionLookup: {a: '1'}, // todo:
+        attribute: 'ss', // todo:
+        cluster: 'genOnOff',
+        commands: [
+            'commandOn',
+            'commandOff',
+            'commandToggle',
+            // 'commandOnWithTimedOff'
+        ],
+    });
+}
+
+function mapModernEnum(cfg: Config, epName: string) {
+    return modernExtend.enumLookup({
+        name: cfg.name,
+        lookup: mapObject(cfg.values, (k) => k.name, (v) => v.value),
+        endpointName: epName,
+        cluster: cfg.cluster,
+        attribute: cfg.attribute,
+        description: cfg.description,
+        access: 'ALL',
+        entityCategory: 'config',
+    });
+}
+
+function addEndpointPrefix(ep: number) {
+    return `${prefixEnpointName}${ep}`;
+}
+
+function mapObject<T>(arr: T[], mapKey: (k: T) => string | [string], mapValue: (v: T) => string | number) {
+    return Object.fromEntries(arr.map((el, i, arr) => [mapKey(el), mapValue(el)]));
+}
+
+const fzLocal = {
+    on_off_action: {
+        cluster: 'genOnOff',
+        type: [
+            'commandOn',
+            'commandOff',
+            'commandToggle',
+            // 'commandOnWithTimedOff'
+        ],
+        convert: (model, msg, publish, options, meta) => {
+            return {action: `${msg.endpoint.ID}_${msg.type.replace('command', '')}`};
+        },
+    } satisfies Fz.Converter,
+};
+
+const definitions: Definition[] = [
+    {
+        zigbeeModel: ['Zucchini'],
+        model: 'Zucchini',
+        vendor: 'Llama Devices',
+        description: 'On/Off output device',
+        fromZigbee: [fzLocal.on_off_action],
+        exposes: [
+            e.action([]),
+        ],
+        extend: [
+            mapModernActionEnum(),
+            modernExtend.battery(),
+            modernExtend.temperature(),
+            modernExtend.humidity(),
+            ...((epCount: number) => {
+                const features: ModernExtend[] = [];
+                for (let ep = 1; ep <= epCount; ep++) {
+                    for (let i = 0; i < configs.length; i++) {
+                        features.push(mapModernEnum(configs[i], addEndpointPrefix(ep)));
+                    }
+                }
+                return features;
+            })(endpointCount),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const ep1 = device.getEndpoint(1);
+            await reporting.bind(ep1, coordinatorEndpoint, ['genPowerCfg']);
```

Comment:
Modern Extends already handle configure inside of them

Reply:
yes, you are right. But, I don't want min 10 seconds, I want 0 seconds for min. Battery handles it, but other hardcoded min: '10_SECONDS'

Reply:
wait a minute, i can do it!
```
modernExtend.battery({
    percentageReporting: true,
    voltageReporting: false,
    percentageReportingConfig: {change: 0, max: 3600, min: 0},
}),
modernExtend.temperature({
    reporting: {min: 0, max: 3600, change: 0},
}),
modernExtend.humidity({
    reporting: {min: 0, max: 3600, change: 0},
}),
```
thanks a lot! configure now smaller :)

P.S. I'm from .NET world. TS for me is a new language